### ISSUE TITLE
Open and Tabopen only search open tabs

### ISSUE THREAD

**Issue by ascenator** [vimperator/vimperator-labs#601]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (X11; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0
```

##### Description:

Before my recent update to FireFox 49 Vimperator worked like a charm. Now, `:open` and `:tabopen` show a weird behaviour: the commands **only** search tabs that are already open, the browser history and bookmarks are **completely** ignored. Thus, their behavior is the same like `:buffer!`.

Additionally - prior to FireFox 49 - `:buffer!` did recognize `%` and `#` for fast switching to the last visited tab. In FireFox 49 this does not work, either. 

##### Expected behavior:

Normal behavior of `:open` and `:tabopen`: taking history and bookmarks into consideration again for any search.

**Comment by timss** [comment]
Thanks for the report.

Sounds like a duplicate of #590. Intended fix merged in #591. Temporary workaround [here](https://github.com/vimperator/vimperator-labs/issues/590#issuecomment-248833125).

**Comment by ascenator** [comment]
Yes, you are right: it is the same issue and workaround works, thanks!
Sorry for the duplicate, must have overseen this issue.
